ios/FluentUI/Card/CardView.swift

```diff
@@ -21,22 +21,138 @@ public enum CardColorStyle: Int, CaseIterable {
     case custom
 }
 
-/// Card style can be horizontal, or vertical
-/// Both styles have a title, an icon and an optional subtitle. The whole card is clickable
-@objc(MSFCardStyle)
-public enum CardStyle: Int, CaseIterable {
-    /// Icon is positioned next to the title and subtitle is positioned below the title
-    case horizontal
-    /// Icon is at the top, title below the icon and subtitle below the title
-    case vertical
+/// Pre-defined sizes of the Card.
+@objc(MSFCardSize)
+public enum CardSize: Int, CaseIterable {
+    case small
+    case large
 
     var width: CGFloat {
         switch self {
-        case .horizontal:
-            return 156
-        case .vertical:
-            return 120
+        case .small:
+            return 164
+        case .large:
+            return 214
+        }
+    }
+    
+    var verticalPadding: CGFloat {
+        switch self {
+        case .small:
+            return 8
+        case .large:
+            return 12
+        }
+    }
+
+    var leadingPadding: CGFloat {
+        switch self {
+        case .small:
+            return 16
+        case .large:
+            return 20
+        }
+    }
+
+    var trailingPadding: CGFloat {
+        switch self {
+        case .small:
+            return 12
+        case .large:
+            return 16
+        }
+    }
+    
+    var baseHeight: CGFloat {
+        switch self {
+        case .small:
+            return 28
+        case .large:
+            return 36
+        }
+    }
+
+    var primaryTextStyle: TextStyle {
+        switch self {
+        case .small:
+            return .caption1
+        case .large:
+            return .subhead
+        }
+    }
+    
+    var secondaryTextStyle: TextStyle {
+        switch self {
+        case .small:
+            return .caption2
+        case .large:
+            return .footnote
+        }
+    }
+
+	func labelHeight(for sizeCategory: UIContentSizeCategory) -> CGFloat {
+        var height: CGFloat = 0
+        switch self {
+        case .small:
+            switch sizeCategory {
+            case .extraSmall:
+                height = 27
+            case .small:
+                height = 27
+            case .medium:
+                height = 27
+            case .large:
+                height = 29
+            case .extraLarge:
+                height = 34
+            case .extraExtraLarge:
+                height = 39
+            case .extraExtraExtraLarge:
+                height = 44
+            case .accessibilityMedium:
+                height = 53
+            case .accessibilityLarge:
+                height = 63
+            case .accessibilityExtraLarge:
+                height = 77
+            case .accessibilityExtraExtraLarge:
+                height = 89
+            case .accessibilityExtraExtraExtraLarge:
+                height = 103
+            default:
+                break
+            }
+		case .large:
```

Comment:
Validation check failed here:
/Users/runner/work/fluentui-apple/fluentui-apple/ios/FluentUI/Card/CardView.swift:125:3: error: Switch and Case Statement Alignment Violation: Case statements should vertically align with their enclosing switch statement. (switch_case_alignment)